quic/s2n-quic-platform/src/io/tokio/builder.rs

```diff
@@ -121,6 +122,15 @@ impl Builder {
         Ok(self)
     }
 
+    /// Disables Generic Receive Offload (GRO)
+    ///
+    /// By default, GRO will be used unless the platform does not support it. If it is known that
+    /// GRO is not available, set this option to explicitly disable it.
+    pub fn with_gro_disabled(mut self) -> io::Result<Self> {
+        self.gro_enabled = Some(true);
```

Comment:
:-)
```suggestion
        self.gro_enabled = Some(false);
```

Reply:
I'm dumb. I initially had the field as `gro_disabled` and forgot to refactor the actual value :laughing: